x/erc20/keeper/msg_server.go

```diff
@@ -186,6 +186,46 @@ func (k Keeper) convertERC20IntoCoinsForNativeToken(
 	return &types.MsgConvertERC20Response{}, nil
 }
 
+// ConvertCoin converts native Cosmos coins into ERC20 tokens for both
+// Cosmos-native and ERC20 TokenPair Owners
+func (k Keeper) ConvertCoin(
+	goCtx context.Context,
```

Comment:
nit: I prefer to just pass the sdk context and not have to unwrap.
I think the unwrapping stuff was part of sdk v2 and I'm not sure about the future best practice here.

Reply:
Yeah there's a bunch all over the repo. I'll create a new issue to go through and switch them all back to sdk.Context